src/components/common/Toast/Toast.module.css

```
@@ -0,0 +1,61 @@
+  position: fixed;
+  bottom: 16px;
+  right: 16px;
+  display: flex;
+  flex-direction: column-reverse; /* 최신 토스트가 아래로 쌓임 */
+  gap: 10px;
+  z-index: 9999;
+  max-width: 90vw;
+}
+
+/* 모바일 반응형 */
+@media (max-width: 480px) {
```

Stylelint (17.2.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)